src/Components/DesignAdjustments.tsx

```diff
@@ -106,3 +154,37 @@ const FontFace = connect(
   },
   { loading: () => null },
 )
+
+function getTextColors(backgroundColor: string): [string, string] {
+  return colorIsLight(backgroundColor)
+    ? ['var(--dark-text-color)', 'var(--dark-headline-text-color)']
+    : ['var(--light-text-color)', 'var(--light-headline-text-color)']
+}
+
+function colorIsLight(hex: string): boolean {
+  if (
+    !/^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{4}|[0-9A-Fa-f]{6}|[0-9A-Fa-f]{8})$/.test(
```

Comment:
* [ ] FMI Does the color picker really store values in the 3 or 4 digits notation?